src/container/srv_target.c

```diff
@@ -448,8 +448,7 @@ cont_vos_aggregate_cb(struct ds_cont_child *cont, daos_epoch_range_t *epr,
 {
 	int rc;
 
-	rc = vos_aggregate(cont->sc_hdl, epr, ds_csum_recalc,
-			   agg_rate_ctl, param, full_scan);
+	rc = vos_aggregate(cont->sc_hdl, epr, agg_rate_ctl, param, full_scan);
```

Comment:
@jgmoore-or & @gnailzenh can confirm, but I believe ds_csum_recalc was inserted because Liang didn't want VOS to be directly dependent on checksum functionality. Maybe that's changed? I'm fine either way.

Reply:
You mean the csum functions in daos_common lib (VOS only need to calls the functions in common lib calculate csums)? I think VOS is dependent on common lib anyway.